src/main/java/org/cbioportal/web/columnar/StudyViewColumnStoreController.java

```diff
@@ -41,16 +73,39 @@
 public class StudyViewColumnStoreController {
     
     private final StudyViewColumnarService studyViewColumnarService;
-    private final StudyViewService studyViewService;
     private final ClinicalDataBinner clinicalDataBinner;
+    private final StudyViewService studyViewService;
+    @Autowired
+    private final ClinicalAttributeService clinicalAttributeService;
+    @Autowired
+    private ClinicalAttributeUtil clinicalAttributeUtil;
+    @Autowired
+    private PatientServiceImpl patientService;
+    @Autowired
+    private StudyViewFilterApplier studyViewFilterApplier;
+    @Autowired
+    private StudyViewFilterUtil studyViewFilterUtil;
+    @Autowired
+    private SampleServiceImpl sampleService;
+    @Autowired
+    private ClinicalDataFetcher clinicalDataFetcher;
+    @Autowired
+    private ClinicalDataDensityPlotService clinicalDataDensityPlotService;
```

Comment:
You don't need @Autowired annotation. Just set it to final and set it via constructor dep injection